src/libsyntax_pos/source_map.rs

```diff
@@ -0,0 +1,1010 @@
+//! The `SourceMap` tracks all the source code used within a single crate, mapping
```

Comment:
Strange... why isn't git recognizing this as a rename of a file? Would be good not to lose blame.